.github/workflows/mobile-i18n-autofill-pr.yml

```
@@ -0,0 +1,100 @@
+            --mode apply \
+            --repo-root "$REPO_ROOT" \
+            --locales "$LOCALES" \
+            --model "$MODEL" \
+            --batch-size $BATCH_SIZE
+
+      - name: Validate Android resources compile
+        run: |
+          ./gradlew :cmp-android:processDemoDebugResources
+
+      - name: Commit and Push changes
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)